src/components/course/EnrollModal.jsx

```diff
@@ -4,14 +4,17 @@ import { Modal } from '@edx/paragon';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSpinner } from '@fortawesome/free-solid-svg-icons';
 
+export const ENROLL_MODAL_TEXT_NO_OFFERS = 'You do not have any applicable vouchers, but you can still enroll in this course.';
```

Comment:
@binodpant I would change this to 'enrollment code' to be consistent.

Reply:
@edx-abolger I did ensure to use the new microcopy changes in this PR too since there was a merge conflict in the EnrollModal.jsx file